test/system_test.py

```diff
@@ -378,15 +555,102 @@ def test_install_version_0_0_4(self):
             'py-icon4py @ 0.0.4 %gcc ^py-gt4py@1.1.1 ^python@3.10.4')
 
 
+class PyInflectionTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-inflection')
+
+
+class PyLarkTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-lark')
+
+
+class PyNumpyTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install('py-numpy')
+
+
+class PyPathspecTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-pathspec')
+
+
+class PyPoetryCoreTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-poetry-core')
+
+
+class PyPytestTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-pytest')
+
+
+class PyPytestFactoryboyTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-pytest-factoryboy')
+
+
+class PySetuptoolsTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-setuptools')
+
+
+class PyToolzTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-toolz')
+
+
+class PyTypingExtensionsTest(unittest.TestCase):
+
+    def test_install_default(self):
+        spack_install_and_test('py-typing-extensions')
+
+
 @pytest.mark.no_tsa  # Irrelevant
 @pytest.mark.no_balfrin  #Irrelevant
 class RttovTest(unittest.TestCase):
 
-    def test_install_version_13_1(self):
-        spack_install_and_test('rttov @ 13.1  %nvhpc')
+    def test_install_version_13_1_gcc(self):
+        spack_install_and_test('rttov @13.1 %gcc')
+
+    def test_install_version_13_1_nvhpc(self):
+        spack_install_and_test('rttov @13.1 %nvhpc')
+
+
+@pytest.mark.no_tsa  # Fails with "C compiler cannot create executables"
+class ScalesPPMTest(unittest.TestCase):
+    # TODO: Add other versions and compilers!
+
+    def test_install_default(self):
+        spack_install_and_test('scales-ppm')
+
+
+class TensorflowCTest(unittest.TestCase):
+    # TODO: Add other compilers!
```

Comment:
No need for another compiler. Downloads a precompiled tar file anyway.

Reply:
```suggestion
```